nixos/modules/module-list.nix

```diff
@@ -633,6 +633,7 @@
   ./services/networking/iperf3.nix
   ./services/networking/ircd-hybrid/default.nix
   ./services/networking/iwd.nix
+  ./services/networking/jitsi-meet.nix
```

Comment:
I was wondering if perhaps services/web-apps might be a better place?

Reply:
Separate PR? Let's get this merged ASAP, it's important these days...

Reply:
Sure, also I was just wondering rather than saying it should go there, as it's a bit of both, e.g. meet is a web-app but videobridge is as I understand more like a server-software which would go into services/networking.